src/docs/release/breaking-changes/target-platform-linux-windows.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Adding `linux` and `windows` to [`TargetPlatform`][] enum
+description: Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][].
+---
+
+## Summary
+
+Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][] and don't include a `default:` case.
```

Comment:
It's sufficient to link on the first instance, or much later in the doc. Remove the second link (but keep the codefonted text).

Also, line length should be less than 80 chars and this link is much longer.

Reply:
OK, fixed.